### DISCUSSION TITLE
px.active_session() to connect to remote endpoint?

### DISCUSSION THREAD

**Discussion by mattcvincent** [Arize-ai/phoenix#2044] [General]
Hi there! When I have phoenix running on a remote machine (e.g. "PHOENIX_COLLECTOR_ENDPOINT" = "https://my-phoenix.io"), how do I create a session to that remote endpoint?

**Comment by mikeldking** [comment]
Hey @mattcvincent very good question. Currently sessions are assumed to not be remote but we will definitely add the ability to attach to remote sessions now that we have remote instances. We are planning on tackling this as part of our persistence story where traces, evaluations, and datasets will be preserved by the remote instance.

CC @RogerHYang

**Reply by mattcvincent** [reply]
Thanks.  In the meantime, is there a way for me to replace

```python
from phoenix.session.evaluation import get_qa_with_reference, get_retrieved_documents
queries_df = get_qa_with_reference(session)
retrieved_documents_df = get_retrieved_documents(session)
```

with some sort of [export](https://docs.arize.com/phoenix/how-to/export-your-data) call to the remote endpoint?

aka `curl https://my-phoenix.io/exports?filename=foo`

If so, can you point me some pointers on how to create the exports of queries and retrieved documents?

**Comment by mikeldking** [comment]
@mattcvincent We launched a client so you can interact with the server easily. https://docs.arize.com/phoenix/api/client

**Reply by mattcvincent** [reply]
Amazing!  I'm going to be sharing Phoenix during an "Intro to LLM Observability" talk at midwest AI user group this month and I'm excited to share this new client during my talk.  Thank you 🙏
